### ISSUE TITLE
NPE in SyncHandler.onPlayerLoggedOut

### ISSUE THREAD

**Issue by ZeroNoRyouki** [specialattack/SpACoreForge#6]
Hi,
in the last couple of days I got 2 server crashes in your SyncHandler.onPlayerLoggedOut method:

java.lang.NullPointerException: Exception in server tick loop
        at net.specialattack.forge.core.sync.SyncHandler.onPlayerLoggedOut(SyncHandler.java:152)
        at cpw.mods.fml.common.eventhandler.ASMEventHandler_875_SyncHandler_onPlayerLoggedOut_PlayerLoggedOutEvent.invoke(.dynamic)
        at cpw.mods.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:54)
        at cpw.mods.fml.common.eventhandler.EventBus.post(EventBus.java:138)
        at cpw.mods.fml.common.FMLCommonHandler.firePlayerLoggedOut(FMLCommonHandler.java:555)
        at net.minecraft.server.management.ServerConfigurationManager.func_72367_e(ServerConfigurationManager.java:318)
        at net.minecraft.network.NetHandlerPlayServer.func_147231_a(NetHandlerPlayServer.java:633)
        at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:154)
        at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
        at net.minecraft.server.dedicated.DedicatedServer.func_71190_q(DedicatedServer.java:334)
        at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
        at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
        at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)

Basically you are not checking if the result of SyncHandler.getConnectionInfo() is null
